rest-service/manager_rest/storage/resource_models.py

```diff
@@ -617,6 +631,25 @@ def make_create_environment_execution(self, **params):
         )
         return self.create_execution
 
+    @property
+    def environment_type(self):
+        for label in self.labels:
+            if label.key == 'csys-env-type':
+                return label.value
+        return ''
+
+    @property
+    def finished_operations(self):
+        if not self.latest_execution:
+            return None
+        return self.latest_execution.finished_operations
+
+    @property
+    def total_operations(self):
```

Comment:
These two _operations properties feel somewhat misleading as they only give you finished/total operations for the most recent execution. I'm unsure if this is a big issue, but it doesn't feel quite right.

Reply:
Well, I added these two properties here because frontend needs to have the latest execution operation for this deployment which will make them to have these values ready to use instead of query each deployment again to get these operations because currently we exposed them,  So I think it would be better to change their names `latext_execution_ total_operations` for example so that we can tell these are for latest execution ?
@geokala

Reply:
Yeah, I think naming them more accurately would probably be good- it's a bit more wordy, but it also says what it actually means.

Reply:
You appear to have removed the default when making them not nullable- I think that'll cause issues with restore?

Reply:
Sure this is back now and part of the migration file